.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[edx-platform.paragon]
```

Comment:
Should this be a separate project, not `edx-platform`?

Reply:
As discussed, we will keep this under `edx-platform` for the time being, as it seems most of the projects that consume Paragon are under this project as well. I agree it's a bit odd, though.